src/app/Library/CrudPanel/Traits/Tabs.php

```diff
@@ -106,56 +106,94 @@ public function isLastTab($label)
     }
 
     /**
+     * @deprecated Do not use this method as it will be removed in future versions!
+     * Instead, use $this->getElementsWithoutATab($this->getCurrentFields())
+     *
      * @return \Illuminate\Support\Collection
      */
     public function getFieldsWithoutATab()
     {
-        $all_fields = $this->getCurrentFields();
+        return $this->getElementsWithoutATab($this->getCurrentFields());
+    }
 
-        $fields_without_a_tab = collect($all_fields)->filter(function ($value, $key) {
+    /**
+     * @return \Illuminate\Support\Collection
+     */
+    public function getElementsWithoutATab(array $elements)
+    {
+        return collect($elements)->filter(function ($value) {
             return ! isset($value['tab']);
         });
+    }
 
-        return $fields_without_a_tab;
+    /**
+     * @deprecated Do not use this method as it will be removed in future versions!
+     * Instead, use $this->getTabItems($tabLabel, 'fields')
+     *
+     * @return array|\Illuminate\Support\Collection
+     */
+    public function getTabFields(string $tabLabel)
```

Comment:
We should also change our code to use the new method. 

AFAIK is only used here: https://github.com/Laravel-Backpack/CRUD/blob/00c96c69a193ee91cd8e2752cf884290724fa779/src/resources/views/crud/inc/show_tabbed_fields.blade.php#L51